### PR TITLE
added sidebar and refactored account page #7268

### PR DESCRIPTION
**Pull request by gautampanday**

### CODE REVIEW

**Comment by gautampanday** [comment]
nereid webshop

**Comment by gautampanday** [comment]
nereid webshop1

**Comment by gautampanday** [comment]
nereid webshop2

**Comment by gautampanday** [comment]
nereid webshop3

File: templates/webshop/account-base.jinja [hunk 1]
```diff
@@ -1,29 +1,43 @@
...
+            <a href="{{ url_for('party.address.view_address') }}">{{ _('Address Book') }}</a>
+          </li>
+        </div>
+        {% if current_user.is_authenticated() and request.nereid_website.save_payment_profile %}
+        <li {% if active == 'my-cards' %}class="active"{% endif %}>
+          <a href="{{ url_for('party.party.view_payment_profiles') }}">{{ _('My Cards') }}</a>
+        </li>
+        {% endif %}
+      </div>
+      {#<li>
```
Comment (GauravButola) [inline templates/webshop/account-base.jinja]: Remove

File: templates/webshop/address-add.jinja [hunk 2]
```diff
@@ -5,21 +5,18 @@
...
+      {{ account_header(_('My Account / Add Address'), 'address-book') }}
+      <div class="col-md-10">
+        <h4 class="account-heading text-center"> {{ _('Add a new Address') }} &nbsp;&nbsp;<small>[ {{ _('Fields marked with (<span>*</span>) are requried.') }}]</small>
+        </h4>
         <form method="post" id="address" autocomplete="off">
-  	{{ form.hidden_tag() }}
-          {{ render_field(form.name, class_="form-control") }}
-          
-          {{ render_field(form.street, placeholder=_("Street address, P.O. box, company name, c/o"), class_="form-control", label_text= _('Address Line 1')) }}
+           {{ form.hidden_tag() }}
```
Comment (GauravButola) [inline templates/webshop/address-add.jinja]: deindent

File: templates/webshop/sales.jinja [hunk 3]
```diff
@@ -6,96 +6,95 @@
...
-                  </td>
-                </tr>
-                {% endfor %}
-              </div>
-              </div>
-              </tbody>
-            </table>
-            <div class="col-md-12" class="pagination">
-            {{ render_pagination(sales, endpoint='sale.sale.render_list', filter_by=request.args.get('filter_by')) }}
+                      {#<span class="text-muted">Size : </span> L#}
```
Comment (GauravButola) [inline templates/webshop/sales.jinja]: Remove

**Pull request merged**